wfe/wfe.go

```diff
@@ -1728,14 +1687,12 @@ func (wfe *WebFrontEndImpl) revokeCertByKeyID(
 		return prob
 	}
 
-	keyID, err := keyToID(key)
+	existingAcct, err := wfe.db.GetAccountByKey(key)
 	if err != nil {
-		return acme.MalformedProblem(err.Error())
+		return acme.MalformedProblem("Cannot obtain key ID from public key")
```

Comment:
Can you also include the `err` in the malformed problem message here?

Reply:
Done.